src/utils.js

```diff
@@ -35,10 +33,11 @@ function bufferToJSON(content: Buffer): Object {
  * @return {Promise<T>}
  */
 function get<T: any>(uri: string, options: any): Promise<T> {
-    const token = options && options.token;
-    token && (headers.Authorization = `token ${token}`);
-    const opts = Object.assign({ headers }, options);
```

Comment:
`options.token` gets added into `opts.headers.Authorization` and `opts.token` here. I'm avoiding a direct pass so we don't accidentally pollute `fetch()` options with our wrapper's options (which may divert in the future).